tasks/system_probe.py

```diff
@@ -52,8 +52,13 @@ def build(ctx, race=False, go_version=None, incremental_build=False):
     ldflags, gcflags, env = get_build_flags(ctx)
 
     # extend PATH from gimme with the one from get_build_flags
-    if "PATH" in os.environ and "PATH" in goenv:
+    if "PATH" in goenv:
```

Comment:
I think this change is unintended (it probably comes from a commit I did in `albertvaka/build-system-probe-omnibus` that I later removed).